babel.config.js

```diff
@@ -32,7 +30,7 @@ const config = {
 		[ '@babel/plugin-proposal-class-properties', { loose: true } ],
 		[ '@babel/plugin-transform-classes', { loose: false } ],
 		[ '@babel/plugin-transform-template-literals', { loose: true } ],
-		isCalypso && [
+		[
```

Comment:
this might have some adverse impact on the sdk? tests seem ok... looks like no one had written a test for code that uses asyncRequire before. @dmsnell @ockham any thoughts on this?